src/lib/actions.js

```diff
@@ -135,3 +135,25 @@ export const collapseSideNav = store.action(() => {
   header.inert = false;
   return {isSideNavExpanded: false};
 });
+
+export const checkIfUserAcceptsCookies = store.action(() => {
+  if (localStorage.getItem("web-accepts-cookies")) {
+    return {
+      userAcceptsCookies: true,
+    };
+  }
+
+  return {showingSnackbar: true, snackbarType: "cookies"};
+});
+
+export const setUserAcceptsCookies = store.action(() => {
+  localStorage.setItem("web-accepts-cookies", 1);
+  return {
+    userAcceptsCookies: true,
+    showingSnackbar: false,
+    // Note we don't set the snackbarType to null because that would cause the
```

Comment:
I think we can solve this in the element too (if we see "type" => null, just hold it for the length of the animation). This is also fine, especially as you've left a comment. 👍